webrtc.html

```diff
@@ -313,6 +313,9 @@ <h2>Dictionary {{RTCIceServer}} Members</h2>
                 {{credential}} is a DOMString, and represents a
                 long-term authentication password, as described in
                 [[!RFC5389]], Section 10.2.</p>
+                <p class="note">To support additional values of
+                {{credentialType}} in future, {{credential}} may need to be
+                re-defined as a union.</p>
```

Comment:
"may evolve in the future as a union" ("may need to be re-defined" sounds like the WG isn't sure yet)